docs/Onboarding.md

```diff
@@ -0,0 +1,47 @@
+# Next Generation Symmetric Apps Onboarding
+
+Welcome to the team! The goal of this document is to get you onboarded and ready to contribute as quickly as possible. 
+
+## Review
+- [Working Agreement](./WorkingAgreement.md)
+- [Bill of Materials](./BoM.md)
+- [Application Architecture](./ApplicationArch.md)
+- [Observability](./Observability.md)
+- [Links, including dashboards](https://github.com/orgs/retaildevcrews/teams/nextgen-team/discussions/3)
+
+## Upskilling
+- k8s upskilling Videos - https://aka.ms/k8s-videos
+- k8s quick start - https://aka.ms/k8sqs
+
+## Overview of Repos
+
+- [ngsa](https://github.com/retaildevcrews/ngsa)
+    - This is our main dev repo
+- [ngsa-app](https://github.com/retaildevcrews/ngsa-app)
+    - This is the repo for the application based on `helium-csharp`
+- [loderunner](https://github.com/retaildevcrews/loderunner)
+    - This is the Load Service repo
+- [akdc](https://github.com/microsoft/kubernetes-developer-cluster-kubeadm)
+    - This script sets up a single-node Kubernetes development cluster on an Azure VM. 
+- [akdc-kind](https://github.com/retaildevcrews/akdc-kind)
+    - This will setup a Kubernetes developer cluster using `Kind` and `GitHub Codespaces`
+
+## Feature Crews
+
+We use Feature Crews to break up themes and epics into pairable tasks. FCs foster more pairing opportunities, allow for faster development, and shorter Dev Crew duration. Each Feature Crew should be cross-Dev Crew to promote collaboration.
+
```

Comment:
How much "process" should a FC adopt?

If it were scrum of scrums (it's not), each scrum team would run full ceremonies. That seems too heavy, but some level of process seems fundamental.